app/assets/javascripts/analytics/error-tracking.js

```diff
@@ -0,0 +1,24 @@
+// Extension to track errors using google analytics as a data store.
+(function() {
+
+    "use strict";
+    var trackJavaScriptError = function (e) {
+      var errorSource = e.filename + ': ' + e.lineno;
+      _gaq.push([
+        '_trackEvent',
+        'JavaScript Error',
+        e.message,
+        errorSource,
+        true //nonInteractive so bounce rate isn't affected
```

Comment:
Good spot, I think the universal analytics syntax half crept in whilst I was writing this.